test/parallel/test-worker.mjs

```diff
@@ -0,0 +1,18 @@
+import { mustCall } from '../common/index.mjs';
+import assert from 'assert';
+import { Worker, isMainThread, parentPort } from 'worker_threads';
+
+const TEST_STRING = 'Hello, world!';
+
+if (isMainThread) {
+  const w = new Worker(import.meta.url);
```

Comment:
Also need a variant of this test that accepts `const w = new Worker(new URL(import.meta.url));`